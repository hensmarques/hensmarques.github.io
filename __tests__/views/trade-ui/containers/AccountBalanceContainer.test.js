import React from 'react';
import AccountBalanceContainer from '../../../../src/views/trade-ui/containers/AccountBalanceContainer';

describe('<AccountBalanceContainer/>', () => {
  it('renders and matches the snapshot', () => {
    const wrapper = shallow(<AccountBalanceContainer />);
    expect(wrapper).toMatchSnapshot();
  });
});
