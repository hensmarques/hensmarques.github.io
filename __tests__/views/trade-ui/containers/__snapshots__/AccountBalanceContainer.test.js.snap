// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AccountBalanceContainer/> renders and matches the snapshot 1`] = `
<WidgetWrapper
  classNames={
    Array [
      "account-balance",
    ]
  }
  tabs={
    Array [
      "Account Balance",
    ]
  }
>
  <Connect(AccountBalance) />
</WidgetWrapper>
`;
